tests/cbmc/proofs/s2n_hash_digest_size/s2n_hash_digest_size_harness.c

```diff
@@ -14,16 +14,15 @@
  */
 
 #include <cbmc_proof/cbmc_utils.h>
-#include <cbmc_proof/proof_allocators.h>
+
```

Comment:
There's a newline here, and at a couple of other places where this header file was removed.